### PR TITLE
Kms 13 kms 14 translations history and set active translation version

### PR DESCRIPTION
**Pull request by MateuszUrbanowicz**

### CODE REVIEW

File: public/views/content/details/tabs/history.html [hunk 1]
```diff
@@ -3,14 +3,96 @@
...
+        <div class="table-box"> <!-- START TRANSLATIONS TABLE -->
+            <table ng-table="tableParams" template-pagination="tablePager" class="table table-striped">
+                <tbody>
+                <tr ng-repeat="translation in $data">
+
+                    <td data-title="'TITLE' | translate" sortable="'email'" class="text-center">
+                        {{translation.title}}
+                    </td>
+                    <td data-title="'CREATED_AT' | translate" sortable="'firstName'" class="text-center">
+                        {{translation.createdAt}}
```
Comment (PSkierniewski) [inline public/views/content/details/tabs/history.html]: createdAt date should be formatted in a more human readable way.

File: public/views/content/details/tabs/history.html [hunk 2]
```diff
@@ -3,14 +3,96 @@
         <div class="page-header no-margin-bottom no-margin-top">
             <h1>
                 {{ 'HISTORY_OF_CHANGES' | translate }}
-                <button ui-sref="content.edit.history({ contentId: content.id, langCode: langCode })"
-                        type="button" class="btn btn-primary pull-right">
-                    {{ 'EDIT' | translate }}
-                </button>
+                -
+                <span class="text-gray">{{ 'LANG_NAME_'+langCode.toUpperCase() | translate }}</span>
```
Comment (PSkierniewski) [inline public/views/content/details/tabs/history.html]: This could be a text-muted class

File: src/app/sass/application.css.scss [hunk 3]
```diff
@@ -189,6 +189,10 @@ li:hover {
   border-color : $brand-danger;
 }
 
+.text-gray{
```
Comment (PSkierniewski) [inline src/app/sass/application.css.scss]: When text-muted will be used, this won't be needed.

File: src/app/src/content/controllers/directives/TranslationDeleteCtrl.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+ * Class DeleteTranslationCtrl
+ *
+ * @package    Admin
+ * @author     Mateusz Urbanowicz <urbanowiczmateusz89@gmail.com>
+ * @copyright  Copyright (c) 2015, Mateusz Urbanowicz
+ */
+
+'use strict';
+
+function DeleteTranslationCtrl($scope, $state, $modal, ContentRepository, Notifications) { // jshint ignore:line
```
Comment (PSkierniewski) [inline src/app/src/content/controllers/directives/TranslationDeleteCtrl.js]: // jshint ignore:line is not needed

File: public/views/content/details/tabs/history.html [hunk 5]
```diff
@@ -3,14 +3,96 @@
...
+
+        <div class="table-box"> <!-- START TRANSLATIONS TABLE -->
+            <table ng-table="tableParams" template-pagination="tablePager" class="table table-striped">
+                <tbody>
+                <tr ng-repeat="translation in $data">
+
+                    <td data-title="'TITLE' | translate" sortable="'email'" class="text-center">
+                        {{translation.title}}
+                    </td>
+                    <td data-title="'CREATED_AT' | translate" sortable="'firstName'" class="text-center">
```
Comment (PSkierniewski) [inline public/views/content/details/tabs/history.html]: The sortable attribute shoud be 'createdAt'

File: public/views/content/details/tabs/history.html [hunk 6]
```diff
@@ -3,14 +3,96 @@
...
     </div>
     <div class="panel-body">
-        {{ 'HISTORY_OF_CHANGES' | translate }}
+
+        <div class="table-box"> <!-- START TRANSLATIONS TABLE -->
+            <table ng-table="tableParams" template-pagination="tablePager" class="table table-striped">
+                <tbody>
+                <tr ng-repeat="translation in $data">
+
+                    <td data-title="'TITLE' | translate" sortable="'email'" class="text-center">
```
Comment (PSkierniewski) [inline public/views/content/details/tabs/history.html]: The sortable attribute shoud be 'title'

File: public/lang/pl_PL.json [hunk 7]
```diff
@@ -96,6 +96,11 @@
     "SEO_DESCRIPTION_DESCRIPTION": "TO jest opis tej strony. Jeżeli zostawisz to pole puste spróbujemy użyć kilku pierwszych zdań treści.",
     "RESTORE": "Przywróć",
     "RESTORE_CONTENT_QUESTION": "Czy napewno chcesz przywrócić tą zawartość?",
-    "CONTENT_HAS_BEEN_RESTORED": "Treść została przywrócona."
-
+    "CONTENT_HAS_BEEN_RESTORED": "Treść została przywrócona.",
```
Comment (PSkierniewski) [inline public/lang/pl_PL.json]: This should be 'Zawartość została przywrócona'

File: public/views/content/details/tabs/history.html [hunk 8]
```diff
@@ -3,14 +3,96 @@
...
+
+                        <div ng-hide="translation.id != ( content.translations | getTranslation:langCode:'id' )">
+                            {{ 'CURRENTLY_ACTIVE_VERSION' | translate }}
+                        </div>
+
+                    </td>
+
+                </tr>
+                <tr ng-hide="$data[0]">
+                    <td colspan="6" class="text-center"><strong>{{ 'NO_ENTRIES' | translate }}</strong></td>
```
Comment (PSkierniewski) [inline public/views/content/details/tabs/history.html]: The colspan value should match the number of table columns.

**Comment by MateuszUrbanowicz** [comment]
@PSkierniewski ptal

File: src/app/src/core/filters/CoreFilters.js [hunk 9]
```diff
@@ -57,4 +57,12 @@ angular.module('CoreFilters', [])
     .filter('trustAsHtml', function($sce) {
         'use strict';
         return $sce.trustAsHtml;
+    })
+
```
Comment (PSkierniewski) [inline src/app/src/core/filters/CoreFilters.js]: Filter docs are missing

**Comment by PSkierniewski** [comment]
:+1:

**Pull request merged**